ui-tests/src/test/java/org/openmrs/reference/page/ClinicianFacingPatientDashboardPage.java

```diff
@@ -80,6 +80,12 @@
     private static final By CONDITIONS_LINK = By.cssSelector(".conditions .info-header i.right");
     private static final By ATTACHMENTS_LINK = By.cssSelector("a[id='attachments.attachments.visitActions.default']");
     private static final By ACTIVE_VISIT = By.cssSelector("visitbyencountertype ul li a");
+    private static final By DELETE_PATIENT = By.id("org.openmrs.module.coreapps.deletePatient");
+    private static final By DELETE_PATIENT_REASON = By.id("delete-reason");
+    private static final By DELETE_PATIENT_CONFIRM = By.cssSelector("#delete-patient-creation-dialog > div.dialog-content > button.confirm");
+    private static final By DELETE_PATIENT_CANCEL  = By.cssSelector("#delete-patient-creation-dialog > div.dialog-content > button.cancel");
+    private static final String REASON = "patient nolonger needed ";
+    
```

Comment:
Minor comment thought: We may get rid of `Line-88` as I see `line-89` already existing to ensure we quality assurance within our code base.

Reply:
Resolved thanks